packages/server/src/tests/epics/epicSelection.ts

```diff
@@ -269,17 +272,27 @@ export const findTestAndVariant = (
         : filterTestsWithoutSuperModePass(priorityOrdered);
 
     if (test) {
-        const variant: EpicVariant = selectVariant(test, targeting.mvtId || 1);
-
         return {
-            result: { test, variant },
+            ...selectEpicVariant(test, banditData, targeting),
             debug: includeDebug ? debug : undefined,
         };
     }
 
     return { debug: includeDebug ? debug : undefined };
 };
 
+function selectEpicVariant(test: EpicTest, banditData: BanditData[], targeting: EpicTargeting) {
+    if (test.isBanditTest) {
```

Comment:
I thought we should change this now to a better name before we go any further ->
https://github.com/guardian/support-analytics/pull/332